test/sandbox/contract/getPastDeposits.ts

```diff
@@ -0,0 +1,30 @@
+import { stableCoinConverterContract } from '../../../src/contracts'
+import Logger from '../../../src/helpers/Logger'
+
+require('dotenv').config()
+
+/**
+ *  SANDBOX: Prints events topics
+ *  RUN:     yarn sandbox test/sandbox/contract/getPastDeposits.ts
+ */
+const log = new Logger('sandbox:contract/getPastDeposits')
+
+async function exec (): Promise<void> {
+  log.info('Get past deposits for contract: %s', stableCoinConverterContract.options.address)
+  const events = await stableCoinConverterContract.getPastEvents('Deposit', { fromBlock: 0, toBlock: 'latest' })
```

Comment:
This thing will eventually run into `Error: Returned error: query returned more than 1000 results`
'member that one?

Reply:
Yep, that's just a sandbox. If that happens, we'll add a starting block or sth :)